modular_zubbers/modules/pda_antagonist/ninja.dm

```diff
@@ -0,0 +1,69 @@
+/datum/dynamic_ruleset/midround/from_living/ninja
+	name = "Crew Ninja"
+	offer_name = "Spider Clan Recruit"
+	midround_ruleset_style = MIDROUND_RULESET_STYLE_LIGHT
+	antag_datum = /datum/antagonist/ninja/crew
+	antag_flag = ROLE_NINJA
```

Comment:
```suggestion
	antag_flag = ROLE_SLEEPER_NINJA
```

Make a new define for this one, keep the override as-is

Reply:
```FAILURE #1: /datum/dynamic_ruleset/midround/from_living/ninja has a non-unique antag_flag Space Ninja (used by /datum/dynamic_ruleset/midround/from_ghosts/space_ninja)! at code/modules/unit_tests/dynamic_ruleset_sanity.dm:40```